Bottom: 15fdf8277096a612c84daf19f8cfdf09d896bc83
Top:    7a44b5300ff317dd0510aea29855afdefcd42515
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-20 20:21:02 -0400

Refresh of spl

---

diff --git a/board/ElecArts/ea3131/ea3131.c b/board/ElecArts/ea3131/ea3131.c
index c88b331..dc09782 100644
--- a/board/ElecArts/ea3131/ea3131.c
+++ b/board/ElecArts/ea3131/ea3131.c
@@ -156,7 +156,9 @@ int board_init(void)
 
 int board_late_init(void)
 {
+#if CONFIG_CMD_NAND
 	prepare_write_nand_params_bbt(1);
+#endif
 }
 
 #ifdef CONFIG_USB_EHCI_LPC313X
